script/build_example_apps

```diff
@@ -58,6 +58,7 @@ readonly OT_OPTIONS=(
     "-DOT_DIAGNOSTIC=ON"
     "-DOT_EXTERNAL_HEAP=ON"
     "-DOT_SLAAC=ON"
+    "-DOT_PING_SENDER=ON"
```

Comment:
This list seems to be sorted alphabetically. Do you want to retain that here?

Reply:
Good eye! Just fixed it. Thanks @jwhui !